kubemarine/resources/schemas/cluster.json

```diff
@@ -0,0 +1,98 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "properties": {
+    "nodes": {
+      "type": "array",
+      "description": "Describe each node of the cluster.",
+      "minItems": 1,
+      "items": {
+        "$ref": "definitions/node.json"
+      }
+    },
+    "node_defaults": {
+      "$ref": "definitions/node_defaults.json"
+    },
+    "cluster_name": {
+      "type": "string",
+      "description": "Address of the cluster on which the Control Plane Endpoint is raised, and used in the calculated parameters."
```

Comment:
Hi @shmo1218,
Here Control Plane Endpoint is raised on address, and address (in other words, the `cluster_name` as parameter) is used in the calculated parameters.
Can't it be now read as Control Plane Endpoint is raised on cluster, and Control Plane Endpoint is used in the calculated parameters? If yes, how to resolve the ambiguity?
Also please note that the description is taken as-is from https://github.com/Netcracker/KubeMarine/blob/feature/yaml_schema/documentation/Installation.md#cluster_name so both places should be the same.

Reply:
Hi, @ilia1243,

I have rolled-back this change to the original statement.